scripts/reindex.py

```diff
@@ -0,0 +1,114 @@
+# purpose of this script is to rebuild a new solr collection
+# it will automatically activate it (by swapping cores)
+
+import os 
+import sys
+import pickle
+import requests
+import time
+from subprocess import PIPE, Popen
+
+
+homedir = os.path.dirname(os.path.dirname(__file__))
+lockfile = os.path.abspath(homedir + '/reindex.lock')
```

Comment:
the command line arg is --rebuild-collection.  should the lock file be rebuild.lock rather than reindex.lock?